Dokumenter hvordan bruke FFE for utviklere som ikke har brukt det før

wkillerud opened the pull request:
WIP for #103 

Dokumentasjonen bør si noe om:

- [x] Hvor man finner dokumentasjonen hvis man kommer hit til Github først
- [x] Kort om hvordan FFE er organisert
- [x] Hvor pakkene kan lastes ned (npm)
- [x] Hvor man finner detaljert dokumentasjon om hver pakke
- [x] Hvordan pakkene brukes, ned på `import`-nivå
- [ ] Mer? :man_shrugging:

wkillerud committed:
docs: Help developers use FFE for the first time

This commit fleshes out the documentation under Kom i gang for utviklere
by explaining a bit about the FFE packages and providing some links. For
convenience and flashiness the commit also introduces the Pakkeplukker
which is a widget to generate an `npm install` command for the most
popular packages with or without React.

wkillerud commented:
Teaser for Pakkeplukkeren (name pending). Så noe liknende i et annet designsystem og synes det ga style points :rocket: :sparkles: 

![screen shot 2018-04-06 at 16 22 07](https://user-images.githubusercontent.com/1223410/38426657-88b613be-39b7-11e8-8a1d-6b72a2fead9c.png)

wkillerud commented:
Re import statements i komponenter - kan vi tweake denne (screenshot, rett under overskriften til hver komponent) så den kopierer import-statement til clipboard? Litt søking i styleguidist-koden ga inntrykk av at det er en [komponent vi kanskje kan overskrive](https://github.com/styleguidist/react-styleguidist/blob/5fbbc6773964042e350d12de8126f325232a48a5/src/rsg-components/Pathline/PathlineRenderer.js)?

![screen shot 2018-04-06 at 16 31 42](https://user-images.githubusercontent.com/1223410/38426907-24c34eca-39b8-11e8-8876-7ac2b1b98930.png)

selbekk commented on src/assets/sds.js:
```diff
@@ -24,4 +24,71 @@ const togglePlay = document.querySelectorAll('.sb1ds-svgcontainer');
+(function npmInstallBuilder () {
+    var bundles = {
+        core: [
+            '@sb1/ffe-core',
```
husk `core-react`

selbekk reviewed (commented):
Utrolig bra jobba @wkillerud ! 

Må importene utkommenteres, eller kan du faktisk bare importere dem?

wkillerud commented:
Imports må utkommenteres, ellers brekker eksempelet :frowning_face: Litt derfor det kanskje hadde vært bedre å få imports i stedet for pathen til fila i den copy-widgeten, om det lar seg gjøre.

Edit: 
Noen andre har tenkt på det samme: 

* https://dialogs.github.io/dialog-web-components/
* https://hallister.github.io/semantic-react/

Skal se om jeg finner ut hvordan de gjorde det :)

Edit edit:
https://github.com/SpareBank1/designsystem/pull/147/commits/621cf0434a6f6db1a0fa3a79605e62001808c59b

wkillerud committed:
docs: Add core-react and grid to Pakkeplukkeren

wkillerud committed:
docs: Replace file path with import statement

The componentPathLine is a small text below the component name in the
styleguide that by default lists the path to the file holding the
component. This commit tweaks the config so that text instead shows the
`import` statement for the component. The text can be copied to the
clipboard at the click of a button.

wkillerud committed:
docs: Reword and add links to key locations

wkillerud commented:
Da tenker jeg denne er i en mergebar tilstand. Si i fra om det er noe jeg skal legge til eller endre :)

selbekk commented:
LGTM!

antidecaf reviewed (dismissed):
Veldig bra!

kwltrs commented on styleguide-content/kom-i-gang/for-utviklere.md:
```diff
@@ -1,3 +1,164 @@
 
-Mer informasjon om hvordan du kommer i gang med utvikling i repositoriet og de forskjellige pakkene finnes i [`README.md`](https://github.com/SpareBank1/designsystem/blob/master/README.md).
\ No newline at end of file
+Komponentene er publisert i forskjellige [pakker på npm under @sb1-scopet av sb1-designsystem](https://www.npmjs.com/search?q=maintainer:sb1-designsystem). Kildekoden til pakkene ligger på [Github](https://github.com/SpareBank1/designsystem/tree/develop/packages) sammen med `README`s med mer detaljerte instruksjoner for hvordan bruke pakkene i prosjektet ditt.
```
Bedre med link til https://www.npmjs.com/org/sb1 isteden av /search?q=maintainer.

wkillerud replied:
Der var den lenken ja, jeg lette etter den :smile:

kwltrs reviewed (commented):
Digger pakkeplukkeren! Mtp vedlikehold, burde den på sikt bli en egen app?

wkillerud committed:
docs: Link to org instead of search by maintainer

wkillerud commented:
> Mtp vedlikehold, burde den på sikt bli en egen app?

Hvis den blir mer kompleks enn den er nå så er det sikkert lurt :) F.eks. hvis vi vil lenke til readmes i webversjonen eller noe sånt. Enn så lenge lever den greit der den er synes jeg.

wkillerud committed:
Merge branch 'develop' into 103-getting-started-as-devs
